api-client/src/endpoints/__tests__/createPOIsEndpoint.spec.ts

```diff
@@ -93,7 +93,7 @@ describe('pois', () => {
   }
   it('should map params to url', () => {
     expect(pois.mapParamsToUrl(params)).toBe(
-      'https://integreat-api-url.de/augsburg/de/wp-json/extensions/v3/locations/?on_map=1'
+      'https://integreat-api-url.de/augsburg/de/wp-json/extensions/v3/locations/?on_map=1',
```

Comment:
Yes, sorry, that's the main change from the latest linting version, adding a trailing comma absolutely everywhere. I do also in general prefer a trailing comma because it makes changes cleaner, though I don't think most of these cases will ever get an additional argument.